playground/nuxt.config.js

```diff
@@ -0,0 +1,7 @@
+import { defineNuxtConfig } from 'nuxt3'
+import colorModeModule from '..'
+
+export default defineNuxtConfig({
+  css: ['~/assets/main.css'],
+  buildModules: [colorModeModule]
```

Comment:
Supposed to be `modules` to follow updated documentation?

Reply:
Indeed you are right.

```suggestion
  modules: [colorModeModule]
```